impl/src/main/java/io/opencensus/impl/internal/DisruptorEventQueue.java

```diff
@@ -131,7 +136,19 @@ private static DisruptorEventQueue create() {
         new DisruptorEnqueuer() {
           @Override
           public void enqueue(Entry entry) {
```

Comment:
We may consider tracing this method too (not suggesting doing it in this PR though).